### DISCUSSION TITLE
There are no objects suitable for adding in the 'Song Select' interface

### DISCUSSION THREAD

**Discussion by nana123126** [ppy/osu#25032] [Ideas] [answered]
The 'Song Select' interface no longer has a place to add objects, unless players can customize the 'Song Select' interface.

What I mean is that it is not appropriate for players to add objects in the 'Song Select' interface because it is already perfect and this editing method may only be applicable to a few people. If you want the 'Song Select' interface to be customizable, the recommended approach is to make the objects self defined, rather than only allowing players to add objects.

![$9W85$BGIF2P16V8_)( H_4](https://github.com/ppy/osu/assets/87159802/23f41f6e-584f-460a-ba1d-48bb6b1ed6aa)
(There is no suitable place to place objects because the star rating is already displayed.)

**Comment by bdach** [comment, marked as answer]
The goal is to make song select elements customisable. The only reason the screen is as barren as it is currently is because that has not yet been implemented.
